SQF/dayz_code/stringtable.xml

```diff
@@ -13403,6 +13412,24 @@
 			<French>Votre humanité est trop %1 ce commerçant refuse de vous parler.</French>
 			<Czech>Tvá lidskost je příliš %1, tento obchodník s tebou odmítá mluvit.</Czech>
 		</Key>
+		<Key ID="STR_EPOCH_ACTIONS_HUMANITY_HIGH">
+			<English>high</English>
+			<German>high</German> -->
```

Comment:
I personally don't like Google Translate. For example, all strings you added for russian localization were incorrect. I think real people who know exact language should translate them. But if you want I can do this for the next update. I'll fix this string in a second.